index.js

```diff
@@ -4,7 +4,8 @@ var util = require('util'),
     winston = require('winston'),
     AWS = require('aws-sdk'),
     cloudWatchIntegration = require('./lib/cloudwatch-integration'),
-    _ = require('lodash'),
+    isEmpty = require('lodash.isempty'),
```

Comment:
What's the benefit of having single functions required rather than the whole library? Is it all about possible future refactorings, for example using native functions instead on lodash's?

Reply:
@lazywithclass Personally, I'd have gotten rid of `lodash` entirely, given that you were only using three functions out of the entire library. But wanted to keep functionality as close to what it is today and not to pollute the PR with further changes.

Reply:
That said, the advantage of requiring single functions is `npm i` time and bundle size (not too critical in node, granted). Importing the entire library just to use `isEmpty` seemed like an overkill to me.

Reply:
Ok fair enough.